pkg/reconciler/broker/broker.go

```diff
@@ -107,11 +107,9 @@ func newReconciledNormal(namespace, name string) pkgreconciler.Event {
 func (r *Reconciler) ReconcileKind(ctx context.Context, b *v1alpha1.Broker) pkgreconciler.Event {
 	// TODO(vaikas): Can we just add this into the controller as part of the filtering
```

Comment:
I was hoping that we could use the:
knative.dev/pkg/reconciler/filter.go and in particular: AnnotationFilterFunc and filter them out before getting here? I think this check is fine safety, but would save other brokers even getting here.
https://github.com/knative/pkg/blob/master/reconciler/filter.go#L21

Reply:
I think we have to do this anyways because trigger updates will enqueue the corresponding broker too. 

https://github.com/knative/eventing/blob/73452a3d820fe3195336ed283d0471faf68c8532/pkg/reconciler/broker/controller.go#L130-L136

Reply:
The filtering is now done by the genreconciler with annotation filter.